test/integration/integration_test.cc

```diff
@@ -521,6 +521,14 @@ TEST_P(IntegrationTest, TestSmuggling) {
     sendRawHttpAndWaitForResponse(lookupPort("http"), full_request.c_str(), &response, false);
     EXPECT_THAT(response, StartsWith("HTTP/1.1 400 Bad Request\r\n"));
   }
+  {
+    std::string response;
```

Comment:
Also, do we have smuggling tests that use HTTP methods that can have body; for example POST.

Reply:
GET can have body in Envoy :-/